### PR TITLE
added login function

### PR DESCRIPTION
**Pull request by Vishnuworld**
- added login function using username and password
-

### CODE REVIEW

File: login.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+USERNAME = "ADMIN"
```
Comment (shetkar1910) [inline login.py]: Can we shift credentials to constant.py

File: login.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+PASSWORD = "admin123"
+
+
+def login(user, password):
+    if user == USERNAME and password == PASSWORD:
```
Comment (Ojaswini08) [inline login.py:6]: Could you please add Docstring

**Review by Ojaswini08 — APPROVED** [review]
Look's Good for me.

**Review by shetkar1910 — APPROVED** [review]
done

**Pull request merged**